nova/templates/configmaps/nova.conf.yaml.j2

```diff
@@ -8,8 +8,6 @@ data:
     vif_plugging_is_fatal = False
```

Comment:
I'll make a single one with conditional configuration block when ironic_compute is enabled